tracee-ebpf/tracee/tracee.bpf.c

```diff
@@ -1307,7 +1307,7 @@ static __always_inline int save_path_to_str_buf(buf_t *string_p, const struct pa
                 bpf_probe_read(&dentry, sizeof(struct dentry*), &mnt_p->mnt_mountpoint);
                 bpf_probe_read(&mnt_p, sizeof(struct mount*), &mnt_p->mnt_parent);
                 bpf_probe_read(&mnt_parent_p, sizeof(struct mount*), &mnt_p->mnt_parent);
-                bpf_probe_read(&vfsmnt, sizeof(struct vfsmount*), (const void*)&mnt_p->mnt);
```

Comment:
The problem here is that mnt_p->mnt is of type `struct vfsmount` and not `struct vfsmount*`